### PR TITLE
Updated  Contributing.md

### PR DESCRIPTION
**Pull request by AnonymousCodes911**
Added lines from 50 to 61, describing proper Contributing etiquettes
Signed-off-by: Utkarsh <backuputk@gmail.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -45,4 +45,18 @@ Before you push to production, make sure to run the following command in order t
...
 ```
 hugo
-```
\ No newline at end of file
+```
+## Contributing guide  
+1. While contributing, make sure you use a separate branch than the Main branch 
+2. If you think you can make some patch to fix the issue, fork the repo, and then make a pull
+ request.
+3. Its always a good practice to squash your commits before creating a final pull reuest, avoid conflicting commits.
```
Comment (oleg-nenashev) [inline CONTRIBUTING.md:53]:
```suggestion
3. It's always a good practice to squash your commits before creating a final pull request, avoid conflicting commits. 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -45,4 +45,18 @@ Before you push to production, make sure to run the following command in order t
...
 hugo
-```
\ No newline at end of file
+```
+## Contributing guide  
+1. While contributing, make sure you use a separate branch than the Main branch 
+2. If you think you can make some patch to fix the issue, fork the repo, and then make a pull
+ request.
+3. Its always a good practice to squash your commits before creating a final pull reuest, avoid conflicting commits. 
+4. If you dont think you can contribute back to the patch/bug, open a new issue
```
Comment (oleg-nenashev) [inline CONTRIBUTING.md:54]:
```suggestion
4. If you don't think you can contribute back to the patch/bug, open a new issue
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -45,4 +45,18 @@ Before you push to production, make sure to run the following command in order t
...
+5. While rasing the issue try to be as detailed(but also specific) as possible to allow others to understand the bug.
```
Comment (oleg-nenashev) [inline CONTRIBUTING.md:55]:
```suggestion
5. When submitting the issue try to be as detailed(but also specific) as possible to allow others to understand the bug.
```

**Review by oleg-nenashev — COMMENTED** [review]
Thanks! There are some typos. Also, DCO needs to be fixed

**Comment by AnonymousCodes911** [comment]
Implemented the changes in the same patch-1, Thank you for the guidance 
Utkarsh Kumar <backuputk@gmail.com>

**Comment by oleg-nenashev** [comment]
The DCO still needs to be fixed to have it merge-able

**Comment by AnonymousCodes911** [comment]
> The DCO still needs to be fixed to have it merge-able

I am sorry for the inconvenience caused I am new to Github contributions, thank you for being patient I 'll update the commit with the DCO soon

**Comment by StackScribe** [comment]
This PR can be closed.  The content was incorporated into https://github.com/keptn/keptn.github.io/pull/1060 .

**Comment by oleg-nenashev** [comment]
Thanks @AnonymousCodes911 ! It was incorporated into the documentation, hence closing it

**Pull request closed without merging**